### PR TITLE
fix warnings in animations.lua

### PR DESCRIPTION
**Pull request by Niklp09**

### CODE REVIEW

File: handlers/animations.lua [hunk 1]
```diff
@@ -7,7 +7,7 @@ function xdecor.sit(pos, node, clicker, pointed_thing)
 	if not top_face(pointed_thing) then return end
 	local player_name = clicker:get_player_name()
 	local objs = minetest.get_objects_inside_radius(pos, 0.1)
-	local vel = clicker:get_player_velocity()
+	local vel = clicker:get_velocity()
```
Comment (SmallJoker) [inline handlers/animations.lua:10]: This either needs a minimal Minetest version bump to 5.4.0 or backwards compat for older versions.

**Comment by Niklp09** [comment]
I see no problem in a minimal Minetest version bump to 5.4.0

**Comment by SmallJoker** [comment]
@Niklp09 in that case, please update `mod.conf` and the README.

**Comment by kilbith** [comment]
This isn't enough to bump to 5.4 as a minimum requirement, many legacy servers use xdecor. Reverted.

**Pull request merged**